styles/components/account/account-user-name.module.scss

```diff
@@ -0,0 +1,93 @@
+.account-user-name {
+	margin-bottom: 20px;
+	position: relative;
+
+	input[type="text"] {
+		overflow: auto;
+		padding-bottom: 3px;
+		font-family: Kanit;
+		width: 275px;
+		height: 55px;
+		position: relative;
+		background-color: transparent;
+		text-align: center;
+		font-size: 3.5rem;
+		font-weight: 500;
+		color: var(--secondary-color);
+	}
+
+	.user-name-input-save {
+		border: none;
+		border-bottom: 4px var(--secondary-color) solid;
+	}
+
+	.user-name-input-edit {
+		border: none;
+	}
+
+	.user-name-view {
+		line-height: 3.5rem;
+		overflow: hidden;
+		text-overflow: ellipsis;
+		text-wrap: nowrap;
+		display: inline-block;
+		padding-bottom: 3px;
+		font-family: Kanit;
+		width: 275px;
+		position: relative;
+		text-align: center;
+		font-size: 3.5rem;
+		font-weight: 500;
+		color: var(--secondary-color);
+	}
+
+	button {
```

Comment:
described in earlier comment but use the component you already made

Reply:
nuh-uh